app/react-native/package.json

```diff
@@ -56,7 +56,7 @@
     "json-loader": "^0.5.4",
     "json5": "^0.5.1",
     "postcss-loader": "^2.0.5",
-    "react-native-compat": "0.0.2",
+    "react-native-compat": "0.0.4",
```

Comment:
Yes - this is currently in a prerelease stage so we'd need to upgrade to `1.x` for precedence options.

Let me update `react-native-compat` to `1.0.0` so we can take advantage of `~`